### PR TITLE
Scrollable list in the geomap popup (#4344)

### PR DESCRIPTION
**Pull request by lhazenoot**
- In release note :
  -  In chapter :  Features
  -  Text : #4344

### CODE REVIEW

File: ui/main/src/app/modules/feed/components/map/map.component.scss [hunk 1]
```diff
@@ -21,6 +21,9 @@
     border-radius: 10px;
```
Comment (quinarygio) [inline ui/main/src/app/modules/feed/components/map/map.component.scss:21]: Could you just fix the copyright header:  Copyright (c) 2022-**2023**, Alliander

**Review by quinarygio — CHANGES_REQUESTED** [review]

**Comment by quinarygio** [comment]
In firefox cards title goes on newline : 
image

**Comment by lhazenoot** [comment]
> In firefox cards title goes on newline : image

@quinarygio Hmm, that is weird! At mine, it just looks normal in firefox. Don't know if this is really a problem? Or can we just leave it for what it is?

Firefox geomap popup

**Comment by quinarygio** [comment]
> > In firefox cards title goes on newline : image
> 
> @quinarygio Hmm, that is weird! At mine, it just looks normal in firefox. Don't know if this is really a problem? Or can we just leave it for what it is?
> 
> Firefox geomap popup

The issue occurs only when there is the scrollbar


> > In firefox cards title goes on newline : image
> 
> @quinarygio Hmm, that is weird! At mine, it just looks normal in firefox. Don't know if this is really a problem? Or can we just leave it for what it is?
> 
> Firefox geomap popup

The problem occurs only when there is the scrollbar

**Comment by quinarygio** [comment]
> > @quinarygio Even with the scrollbar I don't get the issue. Also in Edge and Chrome it works normal. So, I have no clue what it could be. Do you want me to find a solution for it?
> 
> Lets wait for a second check by @freddidierRTE to see if it could be reproduced or if it's just me..

I will merge it as nobody else reproduced the issue

**Pull request merged**